File: CS/MapItemFactory.cs
using DevExpress.XtraMap;

namespace DrawMapItemExample {

    public class MapItemFactory : DefaultMapItemFactory {

        protected override void InitializeItem(MapItem item, object obj) {
            base.InitializeItem(item, obj);
            MapRectangle rect = item as MapRectangle;
            if(rect != null) {
                rect.Width = 1000;
                rect.Height = 1000;
            }
        }
    }
}

File: VB/MapItemFactory.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraMap

Namespace DrawMapItemExample

	Public Class MapItemFactory
		Inherits DefaultMapItemFactory

		Protected Overrides Sub InitializeItem(ByVal item As MapItem, ByVal obj As Object)
			MyBase.InitializeItem(item, obj)
			Dim rect As MapRectangle = TryCast(item, MapRectangle)
			If rect IsNot Nothing Then
				rect.Width = 1000
				rect.Height = 1000
			End If
		End Sub
	End Class
End Namespace
